lib/ramble/ramble/cmd/results.py

```diff
@@ -36,25 +133,56 @@ def results_upload(args):
 
 def import_results_file(filename):
     """
-    Import Ramble experiment results from a JSON file.
+    Import Ramble experiment results from a JSON or YAML file.
+
+    Returns a results dictionary.
     """
     logger.debug("File to import:")
     logger.debug(filename)
 
-    imported_file = open(filename)
+    with open(filename) as imported_file:
+        logger.msg(f"Importing {filename}")
```

Comment:
Maybe update to `Importing results file: {filename}`

Reply:
Done